src/routes/sessions.js

```diff
@@ -62,50 +60,76 @@ module.exports = function (app, opts, dependencies) {
     };
   }
 
+  function getTwoFactorResponse(user) {
+    if (process.env.LIANA_2FA_SECRET  === undefined) {
```

Comment:
So we need a specific `LIANA_2FA_SECRET` environment variable? Why?
`LIANA_` is not the conventional prefix anyway for Forest environment variables.

Reply:
The 2FA user secret sent to the client is the combination of the partial user secret of the server and the `LIANA_2FA_SECRET`. So that, we, at forest, cannot have access to the 2FA user secret.